### ISSUE TITLE
First Name is not truly required when getting users

### ISSUE THREAD

**Issue by dbarkowsky** [bcgov/citz-imb-sso-css-api#1]
The query objects used to search for IDIR users require a firstName field in the documentation, but in practice the Keycloak CSS API doesn't require that field. You can search as long as at least one field in the query parameters (firstName, lastName, email, guid) is populated. 

It would be nice if the functions to get IDIR users would make the firstName field optional as well, making the new type:
``` ts
type IDIRUserQuery = {
  firstName?: string;
  lastName?: string;
  email?: string;
  guid?: string;
};
```

**Comment by dbarkowsky** [comment]
Fixed with this PR: https://github.com/bcgov/citz-imb-kc-css-api/pull/3

**Comment by BradyMitch** [comment]
Resolved in `1.3.1`
